packages/core/src/DdSdkReactNativeConfiguration.tsx

```diff
@@ -18,6 +19,7 @@ export class DdSdkReactNativeConfiguration {
     public site: string = "US"
     public verbosity: NativeSdkVerbosity|undefined = undefined
     public nativeViewTracking: boolean = false
+    public proxyConfig?: ProxyConfiguration = undefined
```

Comment:
why here your are using an optional and above when declaring the `verbosity` parameter you are using `NativeSdkVerbosity|undefined` ? Isn't this the same thing ?

Reply:
yes, it is the same thing, I just didn't want to modify existing code.